app/main/auto-updater/update-checker.js

```diff
@@ -19,4 +19,113 @@ export async function checkUpdate (currentVersion) {
   } catch (ign) { }
 
   return false;
-}
\ No newline at end of file
+}
+
+export function setUpAutoUpdater ({
+  autoUpdater,
+  app,
+  moment,
+  i18n,
+  env,
+  dialog,
+  B
+}) {
+  autoUpdater.setFeedURL(getFeedUrl(app.getVersion()));
+
+  /**
+   * Check for new updates
+   */
+  const checkNewUpdates = async function (fromMenu) {
+    // autoupdate.checkForUpdates always downloads updates immediately
+    // This method (getUpdate) let's us take a peek to see if there is an update
+    // available before calling .checkForUpdates
+    if (process.env.RUNNING_IN_SPECTRON) {
+      return;
+    }
+    const update = await checkUpdate(app.getVersion());
+    if (update) {
+      let {name, notes, pub_date: pubDate} = update;
+      pubDate = moment(pubDate).format(i18n.t('datetimeFormat'));
+
+      let detail = i18n.t('updateDetails', {pubDate, notes: notes.replace('*', '\n*')});
+      if (env.NO_AUTO_UPDATE) {
+        detail += `\n\nhttps://www.github.com/appium/appium-desktop/releases/latest`;
```

Comment:
at this point we should probably change this to `appium-inspector`

Reply:
I agree, but considering that this isn't the only remaining reference to Appium Desktop, I was thinking to make a separate PR for fixing all of them.